src/Codeception/Module/Yii2.php

```diff
@@ -34,6 +34,7 @@
  * * `configFile` *required* - the path to the application config file. File should be configured for test environment and return configuration array.
  * * `entryUrl` - initial application url (default: http://localhost/index-test.php).
  * * `entryScript` - front script title (like: index-test.php). If not set - taken from entryUrl.
+ * * `logTargetsEnabled` - keep log targets enabled or set 'false' to force disable. Default is 'false'
```

Comment:
I'm not so sure that from user's perspective it's clear what are logTargets. We need a better user-firendly name and description. Focus on use case and not on implementation